icons/web3js/web3-plain.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
plain icons should have only a single path. You can read more about our SVG standards on the wiki: https://github.com/devicons/devicon/wiki/SVG-Standards

My suggestion is to base the plain version on the original version, and just make it single-colored. Then you can unify the paths on this icon and call it web3-line.svg